pkg/operator/operator/errors.go

```diff
@@ -260,4 +262,20 @@ func ErrorAPINotDeployed(apiName string) error {
 		Kind:    ErrAPINotDeployed,
 		Message: fmt.Sprintf("%s is not deployed", apiName), // note: if modifying this string, search the codebase for it and change all occurrences
 	})
+
+}
+
+func ErrorRegistryAccountIDMismatch(regID, opID string) error {
+	return errors.WithStack(&errors.Error{
+		Kind:    ErrRegistryAccountIDMismatch,
+		Message: fmt.Sprintf("AWS registry account ID %s doesn't match operator's account ID %s", regID, opID),
```

Comment:
Since we may start calling this from the CLI with local, how do you feel about this:

```go
fmt.Sprintf("registry account ID (%s) doesn't match your AWS account ID (%s), and using an ECR registry in a different AWS account is not supported", regID, opID)
```

Reply:
I like it - mainly because it abstracts the operator, so this could lead to less confusion.